### DISCUSSION TITLE
posts/2024-01-23-cytokines1/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#3] [Announcements]
# posts/2024-01-23-cytokines1/

Immune cells talk to each other through complex networks.

https://rachel.fast.ai/posts/2024-01-23-cytokines1/

**Comment by yvesnewman** [comment]
As I am reading through "The Song of the cell" by Siddhartha Mukherjee and the work of both Mechnikov and Cohen on Phagocytes and Cystokines, I could not be more excited to read about your latest efforts. Fantastic application of AI capability and hopefully making further impact in the are of immune therapy!

**Reply by racheltho** [reply]
Thank you!  Metchnikoff is fascinating-- in addition to his discovery of phagocytes/macrophages, he was also very early in studying the microbiome and probiotics.  "The Song of the Cell" sound interesting, I will have to check it out!
